<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winInternalQuery</class>
 <widget class="QMainWindow" name="winInternalQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Query</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../.designer/backup/icons/InternalQuery.png</normaloff>../../../../.designer/backup/icons/InternalQuery.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 57 12pt &quot;TerminessTTF Nerd Font&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="cbInterval">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>266</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dteStartDate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>266</width>
      <height>34</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>MM/dd/yyyy hh:mm</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dteEndDate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>266</width>
      <height>34</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>MM/dd/yyyy hh:mm</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbDatabase">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>266</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQuery">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>271</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Query Data</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Database</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Interval</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Date/Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>End Date/Time</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listQueryList">
    <property name="geometry">
     <rect>
      <x>328</x>
      <y>128</y>
      <width>461</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddQuery">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>65</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Query</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>328</x>
      <y>0</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Data ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>98</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Query List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemoveQuery">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>560</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Query</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveQuickLook">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Quick Look</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>458</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Quick Look</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbQuickLook">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>271</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoadQuickLook">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Quick Look</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClearQuery">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>520</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Query List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDataIdInfo">
    <property name="geometry">
     <rect>
      <x>382</x>
      <y>5</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/Info.png</normaloff>icons/Info.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIntervalInfo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>76</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/Info.png</normaloff>icons/Info.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbCustomDateTime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Custom</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbPrevDayToCurrent">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous day to current</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbPrevWeekToCurrent">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous 7 days to current</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Date/Time Query Types</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qleDataID">
    <property name="geometry">
     <rect>
      <x>328</x>
      <y>29</y>
      <width>461</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Query Options</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQueryOptionsInfo">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>401</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/Info.png</normaloff>icons/Info.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbDelta">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>424</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Display Deltas</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbOverlay">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>424</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Overlay Pairs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpMax">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/Up-MAX.png</normaloff>icons/Up-MAX.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUp15">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/Up-15.png</normaloff>icons/Up-15.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUp5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/Up-5.png</normaloff>icons/Up-5.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUp1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/Up-1.png</normaloff>icons/Up-1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDown1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>390</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/Down-1.png</normaloff>icons/Down-1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDownMax">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>480</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/Down-MAX.png</normaloff>icons/Down-MAX.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDown15">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>450</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/Down-15.png</normaloff>icons/Down-15.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDown5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>420</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/Down-5.png</normaloff>icons/Down-5.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSearch">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>65</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../.designer/backup/icons/Search.png</normaloff>../../../../.designer/backup/icons/Search.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>qleDataID</zorder>
   <zorder>btnQuery</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>btnAddQuery</zorder>
   <zorder>label_5</zorder>
   <zorder>btnRemoveQuery</zorder>
   <zorder>btnSaveQuickLook</zorder>
   <zorder>label_7</zorder>
   <zorder>cbQuickLook</zorder>
   <zorder>btnLoadQuickLook</zorder>
   <zorder>btnClearQuery</zorder>
   <zorder>btnDataIdInfo</zorder>
   <zorder>btnIntervalInfo</zorder>
   <zorder>cbInterval</zorder>
   <zorder>cbDatabase</zorder>
   <zorder>dteStartDate</zorder>
   <zorder>dteEndDate</zorder>
   <zorder>rbCustomDateTime</zorder>
   <zorder>rbPrevDayToCurrent</zorder>
   <zorder>rbPrevWeekToCurrent</zorder>
   <zorder>label_8</zorder>
   <zorder>label_6</zorder>
   <zorder>label_9</zorder>
   <zorder>cbDelta</zorder>
   <zorder>cbOverlay</zorder>
   <zorder>btnQueryOptionsInfo</zorder>
   <zorder>btnUpMax</zorder>
   <zorder>btnUp15</zorder>
   <zorder>btnUp5</zorder>
   <zorder>btnUp1</zorder>
   <zorder>btnDown1</zorder>
   <zorder>btnDownMax</zorder>
   <zorder>btnDown15</zorder>
   <zorder>btnDown5</zorder>
   <zorder>listQueryList</zorder>
   <zorder>btnSearch</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
